src/data-display/Table/Table.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styles from './Table.module.css'
+import { h } from 'preact'
+
+interface TableCellProps {
+	align: 'left' | 'center' | 'right'
+	children: React.ReactNode
```

Comment:
**nitpick**: You don't have to declare children this way. Let's do `PropsWithChildren<TableCellProps>`